staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -318,6 +312,10 @@ func (v *specStandardValidatorV3) validate(schema *apiextensions.JSONSchemaProps
 		allErrs = append(allErrs, field.Forbidden(fldPath.Child("dependencies"), "dependencies is not supported"))
 	}
 
+	if schema.Ref != nil {
+		allErrs = append(allErrs, field.Forbidden(fldPath.Child("ref"), "ref is not supported"))
```

Comment:
Shouldn't this be `$ref` ?

Reply:
@sttts 

https://github.com/kubernetes/kubernetes/pull/58438#pullrequestreview-89699981

@nikhita 

Is $ref better?

Reply:
It should match the json field